main.go

```diff
@@ -104,3 +105,18 @@ func (s *service) start() error {
 func newService(cfg config.Config) *service {
 	return &service{conf: cfg}
 }
+
+func (s *service) startTechnicalEndpoint(conf config.Config) {
+	log.Printf("Starting technical HTTP endpoint on port: %q", conf.TechnicalEndpointListen)
+	serveMuxHandler := http.NewServeMux()
+	serveMuxHandler.HandleFunc(
+		"/validate/configuration",
```

Comment:
1. IMO it should be /configuration/validate. In the future we may want to add some other actions regarding configuration, like /configuration/update
2. Akubra never terminates connection itself on this port - it sucks.
3. There's no body size limit - we can easily kill akubra uploding too large content.
4. When I run akubra from this branch... well... it opens technical port, that's nice. But it does not open it's ordinary port to handle s3 requests... wtf?